_capabilities/utdallas-lcn.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Learning Interpretable Cutset Networks"
+excerpt: "This library helps the user learn tractable, interpretable cutset networks (a type of probabilistic model which combines decision trees and tree Bayesian networks) from data. The learned networks can be used to answer various decision and explanation queries such as most probable explanation and estimating posterior marginal probability."
+tags: 
+- Analytics
```

Comment:
yaml follows a hierarchical structure, so you need to preserve the two spaces at each level in order for the file to render correctly. You can see the template again [here](https://xaitk.org/contribute/).